build_projects/dotnet-cli-build/GenerateNuGetPackagesArchiveVersion.cs

```diff
@@ -19,6 +19,9 @@ public GenerateNuGetPackagesArchiveVersion()
         }
 
         [Required]
+        public string SDKVersion { get; set; }
```

Comment:
@piotrpMSFT has brought up that the real fix for this is to do a proper evaluation of the template project, so that we can get the implicit package refs as well when generating the hash. I will try to do it tomorrow.